src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -466,6 +467,26 @@ def list_subvolumes(self, **kwargs):
             ret = self.volume_exception_to_retval(ve)
         return ret
 
+    def check_subvolume(self, **kwargs):
+        volname = kwargs['vol_name']
+        groupname = kwargs['group_name']
+        ret = None
+        volume_exists = False
+
+        try:
+            with open_volume(self, volname) as fs_handle:
+                volume_exists = True
+                with open_group(fs_handle, self.volspec, groupname) as group:
+                    res = group.check_subvolumes()
+                    if res:
```

Comment:
IMO we need boolean return values which will tell whether `fs` is okay to delete or not.